flipt-client-react/README.md

```diff
@@ -80,41 +91,29 @@ function MyComponent() {
 }
 ```
 
-This is useful for evalating flags throughtout your application without having to initialize the Flipt client in each component.
+### useFliptSelector Hook
 
-### useFliptClient Hook
+The `useFliptSelector` hook allows the direct access to the Flipt client so that it can be used in a functional component that is wrapped in a `FliptProvider`.
```

Comment:
```suggestion
The `useFliptSelector` hook allows direct access to the Flipt client so that it can be used in a functional component that is wrapped in a `FliptProvider`.
```